src/main/java/interpol/entity/OrderType.java

```diff
@@ -0,0 +1,53 @@
+package interpol.entity;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+@Table(name = "order_type")
+public class OrderType {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(name = "type")
+    private String type;
+
+    @OneToMany(fetch = FetchType.EAGER, mappedBy = "orderType")
+    private Set<OrderDetail> orderDetails;
```

Comment:
Let's remove this field from the class. It looks excessive here.

Reply:
But if we remove this, we won't have link between this table ("OrderType") and table "OrderDetail".

Reply:
In `OderDetail` table you've already has relation to `OrderType` table